avm/res/dev-ops-infrastructure/pool/README.md

```diff
@@ -559,7 +559,7 @@ module pool 'br/public:avm/res/dev-ops-infrastructure/pool:<version>' = {
 | Parameter | Type | Description |
 | :-- | :-- | :-- |
 | [`diagnosticSettings`](#parameter-diagnosticsettings) | array | The diagnostic settings of the service. |
-| [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable/Disable usage telemetry for module. |
+| [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable or disable telemetry for the module. |
```

Comment:
Sorry for the back and forth. This update makes sense, but I'd avoid it since it's a shared parameter, used with same naming and description by all modules.

I'd suggest to revert this change and actually no other change is needed. Since bicep version got updated, we have a change already in main.json and that alone qualifies for publishing

Reply:
Reverted and ran Set-AVMModule. Good to know for next time :)